hack/verify-govet.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+
+GO_VERSION=($(go version))
+
+if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.4') ]]; then
+  echo "Unknown go version '${GO_VERSION}', skipping go vet."
+  exit 0
+fi
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/common.sh"
+source "${OS_ROOT}/hack/util.sh"
+
+cd "${OS_ROOT}"
+mkdir -p _output/govet
+
+FAILURE=false
+test_dirs=$(find_files | cut --delimiter=/ --fields=1-2 | sort -u)
+for test_dir in $test_dirs
+do
+  go tool vet -printf=false \
+              -methods=false \
+              -structtags=false \
+              -composites=false \
+              -buildtags=false \
+              -unreachable=false \
+              -shadow=false \
+              -unusedresult=false \
+              $test_dir 2>&1 | sed '/exit status/d'
```

Comment:
I saw it polluting almost every third line - whenever a `go vet` would fail, the last line would tell you the exit status for whatever reason.